### PR TITLE
Added pressure unit bar

### PR DESCRIPTION
**Pull request by vzorglub**
Added pressure unit bar

### CODE REVIEW

**Comment by htreu** [comment]
Hi @vzorglub once again I have to point you to the Eclipse Contributor Agreement. Please read my comment on your last PR for more details: https://github.com/eclipse/smarthome/pull/5776#issuecomment-399854600

**Comment by vzorglub** [comment]
I was notified of you previous comment, sorry
I have signed the ECA and added Signed-off-by lines to both commits
Live and learn... Thanks a lot

File: bundles/core/org.eclipse.smarthome.core/src/main/java/org/eclipse/smarthome/core/library/unit/SmartHomeUnits.java [hunk 1]
```diff
@@ -167,6 +170,7 @@ public String getName() {
      * @param unit the unit being added.
      * @return <code>unit</code>.
      */
+
```
Comment (htreu) [inline bundles/core/org.eclipse.smarthome.core/src/main/java/org/eclipse/smarthome/core/library/unit/SmartHomeUnits.java]: please remove this empty line.

**Comment by vzorglub** [comment]
I am sorry but what you told me doesn't mean anything to me, I am afraid.

**Comment by htreu** [comment]
No problem, you have to use `git rebase -i ...` in order to fix you previous commit messages (or combine all three commits into one). Please read [this tutorial](https://robots.thoughtbot.com/git-interactive-rebase-squash-amend-rewriting-history) which should address exactly what I'm asking for.
Once you commits are fixed with the correct signed-off-by footer you have to force-push your branch (because you did a rewrite of the history and git would otherwise not allow you to also overwrite the history of the remote repository, which is github in this case).
Use `git push --force`, but be careful and check your history (`git log`) before doing so.

**Comment by vzorglub** [comment]
I am sorry but this tutorial is like reading chinese.
I have no idea what git is nor what does it do.
Can I not start again from scratch and you tell me where I need to write the signed-off-by footer?

**Comment by htreu** [comment]
Hi @vzorglub thanks for not giving up and trying to get this going.
I would be completely okay if you want to start over here with a new PR, too. You did manage to correctly sign off your commits in #5840. Just do it like this.
Are you familiar with programming Java? Otherwise it will be difficult to submit valid code. The lines you proposed here are not compiling, so we can not accept it like this (see also the output of `The Travis CI build failed`).
Another way for you to contribute may be opening concrete issues with detailed analysis and even code snippets like the ones you did here. Another community member is then able to contribute the corresponding java code. what do you think?

**Comment by vzorglub** [comment]
I will try the commit for the Java code again. I have commitments for a
couple of days but I'll be back.
The git command line is not that complicated and there are great guides on
github itself.
My computer is not very powerful and can't handle a dev environment as big
as smarthome is... It was slowing down to a crawl...
I will validate the systax on an online editor
Then I'll use a simple editor on my computer and then
commit with git on the cl.

Thanks for not giving up on me neither.

**Comment by vzorglub** [comment]
Closed and re submitted

**Pull request closed without merging**